tests/unit/workflows/nodejs_npm_esbuild/test_workflow.py

```diff
@@ -244,11 +244,10 @@ def test_workflow_sets_up_esbuild_actions_without_download_dependencies_with_dep
             experimental_flags=[],
         )
 
-        self.assertEqual(len(workflow.actions), 4)
+        self.assertEqual(len(workflow.actions), 3)
```

Comment:
a few tests will have one action less because I moved the action to check esbuild version into the bundling action itself.